cmake/toolchain/sve128.json

```diff
@@ -0,0 +1,3 @@
+{
+  "unit.meta.exe":      "^unit.meta\\..*\\.exe"
```

Comment:
I think it's reasonable to expect that we will have one sve config.

Reply:
True